translation/debug.rst

```diff
@@ -143,7 +143,7 @@ for the ``fr`` locale and run the command, you will get:
     ---------  ------------------  ----------------------  -------------------------------
      State      Id                  Message Preview (fr)    Fallback Message Preview (en)
     ---------  ------------------  ----------------------  -------------------------------
-     missing    Symfony is great    J'aime Symfony          Symfony is great
+     missing    Symfony is great    Symfony is great         Symfony is great
```

Comment:
```diff
  State      Id                  Message Preview (fr)    Fallback Message Preview (en)
  ---------  ------------------  ----------------------  -------------------------------
- missing    Symfony is great    Symfony is great         Symfony is great
+ missing    Symfony is great    Symfony is great        Symfony is great
```

Reply:
To be more clear, it looks like you need to remove 2 spaces before the second occurrence of „Symfony is great“ to match the table, or does it already work?

cc @jaciereguiluz